src/components/frame/v5/pages/FundsPage/partials/Filter/Filter.tsx

```diff
@@ -0,0 +1,315 @@
+import React, { ChangeEvent, ChangeEventHandler, useCallback } from 'react';
+import get from 'lodash/get';
+import set from 'lodash/set';
+import cloneDeep from 'lodash/cloneDeep';
+import { usePopperTooltip } from 'react-popper-tooltip';
+import { AnimatePresence, motion } from 'framer-motion';
+import Icon from '~shared/Icon';
+import Checkbox from '~v5/common/Checkbox';
+import FilterButton from '~v5/shared/Filter/FilterButton';
+import PopoverBase from '~v5/shared/PopoverBase';
+import {
+  RootFilterProps,
+  FilterProps,
+  FilterValue,
+  NestedFilterProps,
+} from './types';
+import { useMobile } from '~hooks';
+import AccordionItem from '~v5/shared/Accordion/partials/AccordionItem';
+import useToggle from '~hooks/useToggle';
+import { accordionAnimation } from '~constants/accordionAnimation';
+import Modal from '~v5/shared/Modal';
+import Header from '~v5/shared/SubNavigationItem/partials/Header';
+import SearchInput from '~v5/shared/SearchSelect/partials/SearchInput';
+
+const displayName = 'v5.pages.FundsPage.partials.Filter';
+
```

Comment:
Should all these functions be in one component?

Reply:
This have to be in one component to prevent from using this outside Filter component